imagevector/images.yaml

```diff
@@ -272,3 +272,16 @@ images:
       confidentiality_requirement: 'low'
       integrity_requirement: 'high'
       availability_requirement: 'low'
+- name: ecr-credential-provider
+  sourceRepository: github.com/gardener/cloud-provider-aws
```

Comment:
I'd have expected this to be `github.com/gardener/ecr-credential-provider` or `github.com/kubernetes/cloud-provider-aws`. Is there a reason to refer to this repository here, as it isn't even indirectly involved with the image?